Gemfile

```diff
@@ -61,5 +61,9 @@ group :test do
   gem 'webdrivers'
 end
 
+group :production do
+  gem 'rails_12factor'
+end
+
```

Comment:
As soon as you have Rails 5+ you don't need this gem. 
https://github.com/heroku/rails_12factor#new-rails-5-apps